specs/deneb/validator.md

```diff
@@ -38,6 +44,15 @@ All behaviors and definitions defined in this document, and documents it extends
 All terminology, constants, functions, and protocol mechanics defined in the updated [Beacon Chain doc of Deneb](./beacon-chain.md) are requisite for this document and used throughout.
 Please see related Beacon Chain doc before continuing and use them as a reference throughout.
 
+## Constants
+
+| Name                  | Value       |
+| --------------------- | ----------- |
+| `ATTESTATION_DUE_MS`  | `6000`      |
+| `AGGREGATE_DUE_MS`    | `9000`      |
+| `SYNC_MESSAGE_DUE_MS` | `6000`      |
+| `CONTRIBUTION_DUE_MS` | `9000`      |
```

Comment:
Ah good idea splitting sync committee and contributions to their own constants, I like it.

Reply:
I think sync messages should be aligned with attestations. the logic is the same as for attestations in that they should be created once the block has been fully validated and it is reasonable to assume that it propagated.